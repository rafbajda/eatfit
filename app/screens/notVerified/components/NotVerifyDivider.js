import React from 'react';
import Divider from 'react-native-divider';
import { lightGrey } from '../../../shared/constants/colors';
import { NotVerifyDividerText } from '../styles/notVerifyDividerStyles';
import I18n from 'i18n-js';

const NotVerifyDivider = () => {
    const { t } = I18n;
    return (
        <Divider borderColor={lightGrey} orientation="center">
            <NotVerifyDividerText>{t('dividers.or')}</NotVerifyDividerText>
        </Divider>
    );
};

export default NotVerifyDivider;
