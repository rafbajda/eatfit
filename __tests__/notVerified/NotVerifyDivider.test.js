import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import NotVerifyDivider from "../../app/screens/notVerified/components/NotVerifyDivider";

describe('NotVerifyDivider snapshot', () => {
    jest.useFakeTimers();
    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it('renders component', async () => {
        const tree = renderer.create(<NotVerifyDivider />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
