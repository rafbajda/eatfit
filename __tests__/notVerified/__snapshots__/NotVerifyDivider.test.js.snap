// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotVerifyDivider snapshot renders component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "height": 24,
      "marginVertical": 6,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "borderBottomWidth": 1,
          "height": 24,
          "transform": Array [
            Object {
              "translateY": -12,
            },
          ],
        },
        Object {
          "borderColor": "#dfe3ee",
        },
        false,
        Object {
          "flex": 1,
        },
      ]
    }
  />
  <Text
    style={
      Array [
        Object {
          "fontSize": 16,
          "fontWeight": "500",
          "paddingHorizontal": 24,
        },
        Object {
          "color": "rgba(0,0,0,.85)",
        },
      ]
    }
  >
    <Text
      className="sc-bdVaJa fuEkbi"
      style={
        Object {
          "color": "#000",
          "fontFamily": "System",
          "fontSize": 16,
        }
      }
      uppercase={false}
    >
      [missing "en.dividers.or" translation]
    </Text>
  </Text>
  <View
    style={
      Array [
        Object {
          "borderBottomWidth": 1,
          "height": 24,
          "transform": Array [
            Object {
              "translateY": -12,
            },
          ],
        },
        Object {
          "borderColor": "#dfe3ee",
        },
        false,
        Object {
          "flex": 1,
        },
      ]
    }
  />
</View>
`;
